app/modules/node.c

```diff
@@ -18,6 +18,18 @@
 
 #define DELAY2SEC 2000
 
+#ifndef LUA_MAXINTEGER
+#define LUA_MAXINTEGER INT_MAX
+#endif
+
+#ifndef LUA_UNSIGNED
+#if LUA_MAXINTEGER > MAX_INT
+typedef uint64_t LUA_UNSIGNED;
+#else
+typedef uint32_t LUA_UNSIGNED;
+#endif
+#endif
```

Comment:
Why this hunk?

Reply:
lua5.1 doesn't have these defines. I guess that we could retrofit them into the 5.1 code as it might make other places a bit simpler.